src/owl/config/configure.ml

```diff
@@ -253,7 +308,7 @@ some details on how your openblas has been installed and the output of
         []
```

Comment:
Change [] to [ "-Wall"; "-pedantic"; "-Wextra"; "-Wunused" ] and remove get_devmode_cflags completely might fix the remaining issue.